bundles/io/pom.xml

```diff
@@ -44,6 +44,7 @@
     <module>org.eclipse.smarthome.io.transport.serial</module>
     <module>org.eclipse.smarthome.io.transport.serial.javacomm</module>
     <module>org.eclipse.smarthome.io.transport.serial.rxtx</module>
+    <module>org.eclipse.smarthome.io.transport.serial.rxtx.rfc2217</module>
```

Comment:
It needs to be added to the [p2 feature](https://github.com/eclipse/smarthome/blob/master/features/org.eclipse.smarthome.feature.runtime.core/feature.xml) as well (and maybe to Karaf, but @maggu2810 is better to comment on that).

Reply:
@msteigenberger Do you need some support for the Karaf feature changes?

Reply:
@maggu2810: yes, would be good if you could give me some pointers

Reply:
If you want an "offline" communication send me an mail or something similar.
If you want to discuss here:
What do you need to know? What does not work? Have you already tried to modify the existing Karaf feature repos (features/karaf/...)?